lib/04_pig_latin.rb

```diff
@@ -0,0 +1,31 @@
+def translate(words)
+  word_string = words.split(" ")
+  vowels = ["a","e","i","o","u","y"]
+  punctuation = [",","!",".","?",";",":"]
+  vowel_index = 0
+  pig_latin = ""
+  for word in word_string
```

Comment:
`for` is rarely used in ruby. Can we use `each`? `for` works; just generally not stylistically common.